streaming/base/format/base/writer.py

```diff
@@ -312,6 +288,7 @@ class JointWriter(Writer):
             while writing). Defaults to ``0``.
         extra_bytes_per_sample (int): Extra bytes per serialized sample (for computing shard size
             while writing). Defaults to ``0``.
+        **kwargs (Any): Additional settings for the Writer
```

Comment:
trailing .

Reply:
Addressed as part of https://github.com/mosaicml/streaming/pull/171/commits/5b5c10450637d4ccdbc8b5c2403a7d18f89b169b.